Content.Shared/Research/Systems/SharedResearchSystem.cs

```diff
@@ -219,16 +220,52 @@ public void TrySetMainDiscipline(TechnologyPrototype prototype, EntityUid uid, T
         if (!Resolve(uid, ref component))
             return;
 
-        var discipline = PrototypeManager.Index<TechDisciplinePrototype>(prototype.Discipline);
+        var discipline = PrototypeManager.Index(prototype.Discipline);
         if (prototype.Tier < discipline.LockoutTier)
             return;
         component.MainDiscipline = prototype.Discipline;
         Dirty(uid, component);
     }
 
+    public bool TryRemoveTechnology(ProtoId<TechnologyPrototype> tech, Entity<TechnologyDatabaseComponent> entity)
+    {
+        return TryRemoveTechnology(PrototypeManager.Index(tech), entity);
+    }
+
+    [PublicAPI]
+    public bool TryRemoveTechnology(TechnologyPrototype tech, Entity<TechnologyDatabaseComponent> entity)
```

Comment:
Maybe just personal preference, but wouldn't it be more consistent to have the Entity<> parameter first on these?